emqx/DOCS.md

```
@@ -1,13 +1,15 @@
 # Home Assistant Community Add-on: EMQX
 
-[EMQX][emqx] is an Open-source MQTT broker with a high-performance real-time
-message processing engine, powering event streaming for IoT devices at massive
-scale. As the most scalable MQTT broker, EMQX can help you connect any device,
-at any scale (including your home).
+[EMQX][https://www.emqx.com/en/products/emqx] EMQX is the world's most
```

markdownlint-cli2 (0.22.1)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "https://www.emqx.com/en/products/emqx"

(MD052, reference-links-images)